test/components/multiselect.spec.js

```diff
@@ -275,7 +278,7 @@ describe('Multiselect', () => {
       });
       await toggle(component);
       await component.get('.search input').setValue('c');
-      const matches = component.findAll('.search-match');
+      const matches = component.findAll('.search-match label');
```

Comment:
`.search-match` now contains the `p.sr-only` element, so we need to select on `label` as well.